pages/rdm_cycle/processing.md

```diff
@@ -1,36 +1,36 @@
 ---
 title: Processing
-keywords:
-contributors: [Rob Hooft]
+keywords: [Data Cleaning, Data Formatting, Quality Check, Data validation]
+contributors: [Rob Hooft, Munazah Andrabi]
 ---
 
 ## What is Data Processing?
 
-Data processing is the phase in a project where data is prepared for analysis.
+Data processing is the phase in the project where data is converted into a desired format and prepared for analysis. When data has been freshly collected, data processing includes some automated steps in a workflow that perform format conversion, quality check and preprocessing following a standardised protocol. The main aim of processing is to:
+ * convert data into readable format giving it the shape and form necessary for downstream analysis 
+ * discard bad or low quality data in order to create clean, high-quality dataset for reliable results.
 
-When data has freshly been collected the data processing can contain some automated steps in a workflow that performs format conversion changes, and preprocessing that is always done in exactly the same way.
-
-Especially when data is imported from elsewhere, e.g. data that is to be reused from another project, processing can also include manual steps to make data suitable for analysis, such as
- * Making changes in data formats to make different data sets that will be integrated compatible with each other
+When data is imported from existing sources, e.g. data to be reused from another project, processing can also include manual steps to make it suitable for analysis. These steps include but are not limited to:
+ * Making changes to data formats such that different data sets will be compatible for integration with each other
```

Comment:
Maybe "datasets" instead of "data sets" is used more often in this website?
Maybe add a full stop at the end of each sentence, even in lists.

Reply:
Done!